### ISSUE TITLE
【ERROR】eavl.py not working

### ISSUE THREAD

**Issue by MuDong612** [tensorflow/models#10714]
Assign requires shapes of both tensors to match. lhs shape= [21] rhs shape= [3]
	 [[node save/Assign_270 (defined at sdb/tmp/users/tongx/zs/models/research/deeplab/eval.py:216) ]]

Errors may have originated from an input operation.
Input Source operations connected to node save/Assign_270:
 logits/semantic/biases (defined at sdb/tmp/users/tongx/zs/models/research/deeplab/model.py:909)

**Comment by purvang3** [comment]
please check your target label shape (number of classes) and your predicted label shape (number of classes). They need to match.

**Comment by chunduriv** [comment]
@MuDong612,

The problem may be due to labels and class numbers that didn't match (or) If you are using any checkpoints from previous runs/versions then try to delete those and train again may resolve above issue. Thank you.
